prac_07/myguitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+import csv
+
+
+def main():
+    guitars = []
+
+    in_file = open('guitars.csv', 'r')
```

Comment:
define 'guitars.csv' as a FILENAME constant. This would allow you to easily pass it to other functions

Reply:
Example, FILENAME = 'guitars.csv'